src/api/app/assets/stylesheets/webui2/coderay.scss

```diff
@@ -0,0 +1,132 @@
+.CodeRay {
+  background-color: #efefef;
+  font-family: Monaco, "Courier New", "DejaVu Sans Mono", "Bitstream Vera Sans Mono", monospace;
```

Comment:
Is this based on the other monospace fonts we use?

Reply:
@hennevogel I adapted to use the `font-family` to exactly what we use with code mirror. I did change the  `background-color` to white.. My first try was similar to github where Comment and Code block have different colors.

Reply:
I re-generated the default stylesheet and now it's using the default `font-stack` (the screenshot was updated). Please take a look @hennevogel